packages/config/tests/cwd/snapshots/tests.js.md

```diff
@@ -20,11 +20,18 @@ Generated by [AVA](https://ava.li).
 
 > Snapshot 1
 
-    `{␊
+    `␊
+    Warning: the "publish" directory was not set and will default to the repository root directory.␊
+    To publish the root directory, please set the "publish" directory to "/"␊
+    To publish the "base" directory instead, please set the "publish" directory to "packages/config"␊
```

Comment:
This warning is unrelated, so it might be good to remove it by using the following `netlify.toml` in the fixture:

```toml
[build]
publish = "/"
```